config.py

```
@@ -5,6 +5,37 @@
+        gpu_available = True
+        gpu_memory_gb = torch.cuda.get_device_properties(0).total_memory / (1024**3)
+    
+    # Check for Apple Metal (MPS)
+    elif hasattr(torch.backends, 'mps') and torch.backends.mps.is_available():
+        gpu_available = True
+        # Estimate Metal memory (simplified)
+        gpu_memory_gb = 8.0  # Conservative estimate for Apple Silicon
+    
+    return gpu_available, gpu_memory_gb
+
+def _detect_system_memory():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)